server/app.js

```diff
@@ -15,6 +18,18 @@ app.get('/listinginfo', (request, response) => {
     response.send(result).status(200).end();
   });
 });
+app.put('/listinginfo', (req, res) => {
+  db('put', (err, result) => {
+    if (err) throw err;
+    res.status(200).end();
+  }, req.body);
+});
+app.delete('/listinginfo', (req, res) => {
+  db('delete', (err, result) => {
+    if (err) throw err;
+    res.status(200).end();
+  }, req.body);
+});
 
```

Comment:
Looks okay. But, again, it'll be important that we send back error codes rather than throwing errors. We will use them for calculating error rates under load tests.